include/uv-darwin.h

```diff
@@ -52,6 +57,7 @@
   void* cf_member[2];                                                         \
   int cf_error;                                                               \
   uv_mutex_t cf_mutex;                                                        \
+  enum uv_fs_event_type_flag type_flag;                                       \
```

Comment:
That's nice to know. I'm not sure this justifies using those fields though? I mean, this would be nice since OS X typically has such a low max open file limit (256 by default), but that is yall's call.